simpleflow/swf/process/decider/base.py

```diff
@@ -72,13 +72,11 @@ def __init__(self, executors, domain, task_list, nb_retries=3,
 
         self.nb_retries = nb_retries
 
-        Poller.__init__(
-            self,
-            domain,
-            task_list,
-            *args,    # directly forward them.
-            **kwargs  # directly forward them.
-        )
+        # Call super()'s methods: as this class uses multiple inheritance, the
+        # call is written directly so it's easier to understand.
+        # TODO: see if *args/**kwargs forwarding is really useful
+        Poller.__init__(self, domain, task_list, *args, **kwargs)
+        swf.actors.Decider.__init__(self, domain, task_list)
```

Comment:
"of course": well, `super` only follows MRO and so it calls only the `__init__()` method of the first class at left. Hence other mixins initialization are not properly called, and that makes me anxious. Am I wrong? :( Maybe in that case we can explicitly document that we don't call `Decider.__init__()` for that reason...